<launch>
	<node name="vector_ros" pkg="anki_vector_ros" type="anki_vector_core.py" output="screen" />
</launch>